updater.js

```diff
@@ -0,0 +1,225 @@
+var path = require('path')
+var fs = require('fs')
+var childProcess = require('child_process')
+var pkgUp = require('pkg-up')
+
+var BrowserslistError = require('./error')
+
+var PACKAGE_CANIUSE = 'caniuse-lite'
```

Comment:
Let’s remove this constant. I do not know a reason to have have `caniuse-lite` directly in strings and code

Reply:
If you plan to use a package with a different name, such as `caniuse-lite-json`, then I should leave it and not inline it is code.

Reply:
Rare case. In this case I will replace it with Find-and-Replace in my text editor.